deepmd/deepmd_property_tools/deepmd_property_tools/predictor.py

```
@@ -0,0 +1,100 @@
+            self.save_predict(rows, y_pred, Path(save_path), prefix)
+        return y_pred
+
+    def standardize(
+        self, atoms: list[list[str]], coordinates: list[np.ndarray]
+    ) -> tuple[np.ndarray, np.ndarray]:
+        if not atoms:
+            raise ValueError("No samples to predict")
+        max_natoms = max(len(symbols) for symbols in atoms)
+        coords = np.zeros((len(atoms), max_natoms, 3), dtype=np.float32)
+        atom_types = np.full((len(atoms), max_natoms), -1, dtype=np.int32)
+        for frame_idx, (symbols, coord) in enumerate(zip(atoms, coordinates)):
```

Ruff (0.15.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)